packages/demo-wallet-client/src/components/KycFieldInput.tsx

```diff
@@ -28,8 +32,31 @@ const getInputType = (type: KycFieldType) => {
   }
 };
 
-export const KycFieldInput = ({ id, input, onChange }: KycFieldInputProps) => {
-  const label = `${input.description}${input.optional ? " (optional)" : ""}`;
+export const KycFieldInput = ({
+  id,
+  input,
+  onChange,
+  isRequired,
+}: KycFieldInputProps) => {
+  const _isRequired = isRequired || !input.optional;
```

Comment:
what are the differences between `isRequired` and `input.optional`? Is there instances when they are out of sync? My understanding is the `!input.optional === isRequired`, but maybe that's not the case

Reply:
I should have clarified this. `isRequired` flag overrides `input.optional`, which we get from the API. We allow optional fields when they are initially required in KYC, but we make them all required when they are in `required_customer_info_updates` field requested later.